app/policies/target_policy.rb

```diff
@@ -0,0 +1,20 @@
+class TargetPolicy < ApplicationPolicy
+  class Scope
+    def initialize(user, scope)
+      @user  = user
+      @scope = scope
+    end
+
+    def resolve
+      scope.all
+    end
+
+    private
+
+    attr_reader :user, :scope
+  end
+
+  def create?
+    true
```

Comment:
I think this is not a good rule, at least we should check that is a logged in user